inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.cpp

```diff
@@ -21,79 +24,86 @@ namespace {
 using DataType = ngraph::element::Type_t;
 using DataDims = ngraph::Shape;
 
-struct NonMaxSuppressionTestCase {
-    int64_t num_batches, num_boxes, num_classes, max_output_boxes_per_class;
-    float iou_threshold, score_threshold;
+// tests are not capable to validate the functions with multiple outputs
```

Comment:
Is it because comparator works only with single-output functions? I guess it's time to enhance it :)

Reply:
I agree :)
But I suggest to do it in a separate request (I've created the ticket on this #-41403 and planning to do this right after NMS-5 will be done) since there are too much tests affected.

Reply:
You can improve comparator in this request and update only new tests. What do you think?

Reply:
That's a good idea, done.